pkg/manifest/v1alpha/objects.go

```diff
@@ -108,8 +126,26 @@ type Calendar struct {
 	TimeZone  string `yaml:"timeZone" validate:"required,timeZone" example:"America/New_York"`
 }
 
+// Metadata represents part of object which is is common for all available Objects, for internal usage.
```

Comment:
Did you consider having these "common" structs at a level outside of a specific version so they don't need to be duplicated?

Reply:
I did, but since they can change from version to version, for example here the `Metadata` changed in v1 to include `labels` and `annotations`, that I think its better to keep it in a specific version.

Reply:
That's fair, thanks